res/controllers/Pioneer-DDJ-SX-scripts.js

```diff
@@ -1114,8 +1117,9 @@ PioneerDDJSX.slicerButtons = function(channel, control, value, status, group) {
     }
 
     if (PioneerDDJSX.activeSlicerMode[deck] === PioneerDDJSX.slicerModes.contSlice) {
-        engine.setValue(group, "beatloop_" + PioneerDDJSX.selectedSlicerQuantization[deck] + "_activate", value);
         engine.setValue(group, "slip_enabled", value);
+        engine.setValue(group, "beatloop_size", PioneerDDJSX.selectedSlicerQuantization[deck]);
+        engine.setValue(group, "beatloop_activate", value);
```

Comment:
This change wasn't necessary because beatloop_X_activate changes beatloop_size to X, but there's no need to change it back.